domains/technosaqib.json

```
@@ -0,0 +1,9 @@
+{
+  "name": "technosaqib",
+  "owner": {
+    "email": "saqibsarwar003@gmail.com"
```

username should be inside `owner` and should match github username
```suggestion
  "owner": {
    "username": "Saqibsarwar12",
    "email": "saqibsarwar003@gmail.com"
```